doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -779,6 +779,13 @@ This section provides detailed lists of changes by :ref:`script <scripts>`.
     * Support for devices using MCUboot bootloader built in the direct-xip mode (``MCUBOOT+XIP``).
       In this mode, the image is booted directly from the secondary slot without moving it to the primary slot.
 
+* :ref:`west_sbom`:
+
+  * Changed:
+
+    * Reduce RAM usage. The script run `Scancode-Toolkit`_ detector in a single process which singificantly reduces memory usage.
+      This change slower licenses detector because it is not executed parallel on files.
```

Comment:
```suggestion
    * To reduce RAM usage, the script now runs the `Scancode-Toolkit`_ detector in a single process.
      This change slows down the licenses detector, because it is no longer executed simultaneously on all files.
```

Reply:
Done